### PR TITLE
[DOCS] Adds DFA limitation regarding feature importance

### PR DESCRIPTION
**Pull request by szabosteve**
Adds a new item to the DFA limitation section about possible performance effects of feature importance calculation.

Related issue: https://github.com/elastic/ml-team/issues/274#issuecomment-585262454

### CODE REVIEW

File: docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc [hunk 1]
```diff
@@ -130,3 +130,17 @@ of combinations of dot delimited and nested fields (for example:
 `{"a.b": "c", "a": {"b": "c"},...}`), the performance of the operation might be 
 slightly slower. Consider using as simple mapping as possible for the best 
 performance profile.
+
+[float]
+[[dfa-feature-importance-limitation]]
+=== Analytics runtime performance may slow down with feature importance
+
+For complex models – those with deep trees and many leaves – the calculation of
```
Comment (valeriy42) [inline docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc]:
```suggestion
For complex models – those with many deep trees – the calculation of 
```

File: docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc [hunk 2]
```diff
@@ -130,3 +130,17 @@ of combinations of dot delimited and nested fields (for example:
 `{"a.b": "c", "a": {"b": "c"},...}`), the performance of the operation might be 
 slightly slower. Consider using as simple mapping as possible for the best 
 performance profile.
+
+[float]
+[[dfa-feature-importance-limitation]]
+=== Analytics runtime performance may slow down with feature importance
+
+For complex models – those with deep trees and many leaves – the calculation of 
+feature importance takes extra time. Feature importance is calculated at the end
```
Comment (valeriy42) [inline docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc]:
```suggestion
feature importance takes significant extra time. Feature importance is calculated at the end 
```

File: docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc [hunk 3]
```diff
@@ -130,3 +130,17 @@ of combinations of dot delimited and nested fields (for example:
...
+[[dfa-feature-importance-limitation]]
+=== Analytics runtime performance may slow down with feature importance
+
+For complex models – those with deep trees and many leaves – the calculation of 
+feature importance takes extra time. Feature importance is calculated at the end 
+of the analysis and therefore the job may appear to be stuck at 99% for several 
+hours.
+
+If a reduction in runtime is important to you, please try strategies such as 
+disabling feature importance, using a smaller training percent, setting
```
Comment (valeriy42) [inline docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc]:
```suggestion
disabling feature importance, using a smaller transform, setting 
```

File: docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc [hunk 4]
```diff
@@ -130,3 +130,17 @@ of combinations of dot delimited and nested fields (for example:
 `{"a.b": "c", "a": {"b": "c"},...}`), the performance of the operation might be 
 slightly slower. Consider using as simple mapping as possible for the best 
 performance profile.
+
+[float]
+[[dfa-feature-importance-limitation]]
+=== Analytics runtime performance may slow down with feature importance
```
Comment (valeriy42) [inline docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc]:
```suggestion
=== Analytics runtime performance may significantly slow down with feature importance computation
```

**Comment by szabosteve** [comment]
@valeriy42 Thanks for the suggestions, I applied them.

**Comment by szabosteve** [comment]
**Backport commits:**
7.x: 5addaf0
7.6: ff759e5

**Pull request merged**